<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAN_BUS_IHM</class>
 <widget class="QMainWindow" name="CAN_BUS_IHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLCDNumber" name="luminosite_display"/>
    </item>
    <item row="1" column="1" colspan="7">
     <layout class="QVBoxLayout" name="layout_open_gl"/>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="luminosite_label">
      <property name="text">
       <string>Luminosité</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLCDNumber" name="distance_display"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pression</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Angles :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radian_button">
        <property name="text">
         <string>Radians</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="degree_buttton">
        <property name="text">
         <string>Degrés</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PHI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="phi_display"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>THETA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="theta_display"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PSI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="psi_display"/>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="distance_label">
      <property name="text">
       <string>Distance (cm)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLCDNumber" name="pression_display"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
